# Bazaar merge directive format 2 (Bazaar 0.90)
# revision_id: serkan@gentoo.org-20081014204742-m1f1undpfdbu4ikc
# target_branch: ../mainline/
# testament_sha1: 8cd8d12ef2cb826a26b67f1550386f4dcc9e73bb
# timestamp: 2008-10-15 00:11:58 +0300
# base_revision_id: andrew@operationaldynamics.com-20081014015640-\
#   81xaue23uz6g5bce
# 
# Begin patch
=== modified file 'Makefile'
--- Makefile	2008-07-13 05:10:28 +0000
+++ Makefile	2008-10-14 20:47:42 +0000
@@ -61,7 +61,7 @@
 
 install: all \
 		$(DESTDIR)$(PREFIX) \
-		$(DESTDIR)$(PREFIX)/share/java/slashtime-$(APIVERSION).jar \
+		$(DESTDIR)$(JARDIR)/slashtime-$(APIVERSION).jar \
 		tmp/stamp/install-pixmaps \
 		$(DESTDIR)$(PREFIX)/share/applications/slashtime.desktop \
 		$(DESTDIR)$(PREFIX)/bin/slashtime
@@ -74,7 +74,7 @@
 	@echo -e "MKDIR\t$@/"
 	-mkdir -p $@
 
-$(DESTDIR)$(PREFIX)/share/java:
+$(DESTDIR)$(JARDIR):
 	@echo -e "MKDIR\t$@/"
 	-mkdir -p $@
 
@@ -111,11 +111,13 @@
 	cp -f share/pixmaps/*.png $(DESTDIR)$(PREFIX)/share/pixmaps
 	touch $@
 
-$(DESTDIR)$(PREFIX)/share/java/slashtime-$(APIVERSION).jar: \
-		$(DESTDIR)$(PREFIX)/share/java \
+$(DESTDIR)$(JARDIR)/slashtime-$(APIVERSION).jar: \
+		$(DESTDIR)$(JARDIR) \
 		tmp/slashtime.jar
 	@echo -e "INSTALL\t$@"
 	cp -f tmp/slashtime.jar $@
+	@echo -e "SYMLINK\t$(@D)/slashtime.jar -> slashtime-$(APIVERSION).jar"
+	cd $(@D) && rm -f slashtime.jar && ln -s slashtime-$(APIVERSION).jar slashtime.jar
 
 
 # --------------------------------------------------------------------

=== modified file 'configure'
--- configure	2008-10-14 01:51:05 +0000
+++ configure	2008-10-14 20:47:42 +0000
@@ -343,6 +343,7 @@
 # --------------------------------------------------------------------
 
 my $prefix;
+my $jardir;
 my $compiler;
 my $runtime;
 my $jdk_home;
@@ -375,6 +376,9 @@
 	} elsif (($key =~ /^--prefix$/) ||
 		 ($key =~ /^prefix$/)) {
 		$prefix="$value";
+	} elsif (($key =~ /^--jardir$/) ||
+		 ($key =~ /^jardir$/)) {
+		$jardir="$value";
 	} elsif ($key =~ /^runtime/) {
 		$runtime="$value";
 	} elsif ($key =~ /^compiler/) {
@@ -394,6 +398,10 @@
 	$prefix="/usr/local";
 }
 
+if (!$jardir) {
+	$jardir=$prefix . "/share/java";
+}
+
 
 # check jdk_home override. compiler and runtime are checked 
 # later (at the end) against choices that have been validated.
@@ -1340,6 +1348,7 @@
 
 print CONFIG "APIVERSION=$apiversion\n";
 print CONFIG "VERSION=$version\n";
+print CONFIG "JARDIR=$jardir\n";
 
 # --------------------------------------------------------------------
 # Done! Create .config file

# Begin bundle
IyBCYXphYXIgcmV2aXNpb24gYnVuZGxlIHY0CiMKQlpoOTFBWSZTWZBH8EYAAvHfgCAwdXP//3//
//W/////YAZvr3tygAJ7bZlpQAIJREAKeVP0J6p4FP00CmnmqbUGj00j1BpkD1PKA3qgJSU8qbFM
1HjQSZNGQ9TRkDQ0AZDE0AGgaFT1NJnqj1D1AGgDQA0GIAAAAAAFU/KntU0GgAYjQAABoNBoAAA0
AHMJoyNDQyGEaGQ00aADEZMgGEAwCSQIBpGQFMypsaU2ymjUeTRpNMIGIBpoaaRiNBoKFm2ewtx3
P7eofvMxkX3papK5TwhOtKcE7mKWLG+5RTKDeqTm5Ku8dbUAjE0XNzQvrg0YxrlRcqcjZsni/Hss
vW+z41Lalgfv8ONmRP8XDlFEVUVR3giP/x2RVW6t6pmEDk3MICKpzDOEAI4zQcAg2EWqZrCKDy+R
DlMWdoCqTLLmKutGA6ZTyFPSRhslUBJMESLARscwt1xEpuzgiBxIFBF8hbi06HY9NWIcPA3OUzsc
6ehUCeDU89hffVUQW70BWc5QGc7aIlOkbXtInExFjyiGYG6vF0e2c+54NwBxFt2GItYMC9Serv4u
NetmpDrFmEQChbLDrEcmP9OLX6nWPEJnE5E76KupEL4+m9TK8u7AoNtc7CqNObTcSMSysvC7Cs4B
Hzae0sKQQxUySoFWTH6h53h5UUkiioeIuJRK3lQxSouEGYqT8KCRId5b/XMcgpoax8CfmLyhJS9g
2jCouGGawYxjLLRkxM5Oogj6kwix415NdhFEGac/J33OqvkWSZfHcIrPj3rhGgLCvRakfLVA1gjQ
REQM44jA0hDUXmFurVrN0tgxNFwtt5AeXE3mgupYeOsKM5KI6UGTQHEBuKY6/VE7S4JyuI6nBaiU
XKBqrEPEUF5KJdocxEFSZVlMaiA8+QifknkfeDplr0rBxNn7cpAiTNRAaqgJmeyveGask3/FmTU5
1t7aGMpOWU9CLrWGD+5TpNgfd4qDvEtmn5hDFVqF+URVlyGLayoyItsKDFrSodAHmYg9JcgUj4Eh
hzh5L3+pxs75EuP+Mkt7WcY40EcxHlMDOaAvOmvo4+GFRFtCLyoMUQsURR3VMfSeRc7JeViOvXMc
UDnQvcMZRh2xyC6SbU1VtgYRVNa56jc8iFYm/Npg/UAcUN352my3xh5S0oqC/Qhv477YYhkfEmB7
9goZl1rZKyDYTdwdPaHtecv5fTDHGMRuckleJ6stIz7oBAgOg8E8O5C+287btX6MzWXuzHMHsxNL
Pg7yuDoeu6lcQafd+dKDbFDDWhZ/vHEGUHULQ+6CmrBs6uYzTAvU5rw8kcmfRJOno7shk8gooiru
lycwYKbETUfGxkhlxEFK8b2EoEWA7Wkol0xkwnZaaHpzCbfxMkU+08DL3o7WGYRx5kqfT7kF6rFd
tCr0d/wPPv9JrcZOED2ofmrkIrM4UoTC5i057DnGlO3yuzEx1wmYcl9KpRVJCz5CYiitXBHqNFtj
Ri+EutjJAahOMGkIlDwohh1uRD1ZBlTfUA0AZHlE7WN0nNhF79cQePHAw4zHMXEygOR2920v8Rar
PcOzXFCmDBLetirR03IZGJPKuubQL19CJSISoxErOyN0xM3j7GSKvwaIdnKTKmuAL3iyEInFyc5+
QJ6wOU3Q5ev19MirARuVy+fKH6l4eEzjoODtKH6UjIDtwGa0iGyNd9lBa4bPH/MWBo+S1iDUQsqm
aRWBLGZBjg5TmBqgqNghwiSRq5RUtoeV/p3bKI21MIYTdJMCHgEYX30E7/MFLBifD4I8+w8i06zz
GOsFIRzBUcQod2XBFjIKBUgcGI+g0xXnOGjh4KSUztFqEQ6tX2SV6CkIczx4r+ux/Quem2mMhvk4
1VOCCR+DhGKgLjjSLDQIeKgUIMybOyZJMyX5uCW7u3qlI7dNfMlUIPOzgoSIvOwTgywF0JHVcF3b
FToY4/c/i27URUhHUIqz7+/VUN76ffxec2I1MmTDxnDmIi3vFABrQc3TN6VtxgTZPyCsSK6tdbwU
KGDPKIRUGGY3jlftnHYL63VjYVv6TX/eKycAcWxWfBQlFBwjlWZkmoLUUUsMwzXnjGLEn2Qpbw+o
3zkewOPEHE7ESNdpzIIGisTlNIrEwpMjGPrgahH0Ett1Ac51CLbcQy7a8TrCiP7YW/T2VLS3GIzj
CZ/w9VxwNBTiJCVDJcYTPxqpDq17FSqiUQelIEpRGZU+T6P8XckU4UJCQR/BGA==
